locale/fi/icons.json

```diff
@@ -1,9 +1,14 @@
 {
   "title": "Ikonit (Icons)",
   "intro": "Tältä sivulla löydät kaikki ikonit jotka ovat käytössä suomi.fi palveluissa. Ikonien käyttöohjeistukset löydät ikonien komponentti sivulta.",
+  "iconPageLink.title": "Ikoni komponentti käyttöohjeistus",
   "baseIcons.title": "Perusikonit (Base icons)",
   "illustrativeIcons.title": "Kuvitusikonit (Illustrative icons)",
   "doctypeIcons.title": "Liitetiedosto-ikonit (Doctype icons)",
+  "baseIcons.link.title": "Perusikonit Githubissa SVG muodossa",
+  "illustrativeIcons.link.title": "Kuvitusikonit Githubissa SVG muodossa",
+  "doctypeIcons.link.title": "Liitetiedosto-ikonit Githubissa SVG muodossa",
```

Comment:
Probably would be correct to use the `<abbr>` tag for the "SVG" text, but with the current localization system might not be doable.

Reply:
Seems that there is no easy way to add that.

Reply:
Yeah, let's leave it as it is for now.